pkg/sfu/streamallocator.go

```diff
@@ -892,12 +894,13 @@ func (s *StreamAllocator) allocate() {
 	// So, track total requested bandwidth and mark DEFICIENT state if the total is
 	// above the estimated channel capacity even if the optimal signal is true.
 	//
+	// LK-TODO make protocol friendly structures
```

Comment:
@davidzhao Made this note to change the type, but brain is fried from all the other stuff :-) Will take care of this in a subsequent PR.